genesyscloud/external_contacts_organization/resource_genesyscloud_externalcontacts_organization_schema.go

```diff
@@ -0,0 +1,366 @@
+package external_contacts_organization
+
+import (
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	"terraform-provider-genesyscloud/genesyscloud/util"
+	"terraform-provider-genesyscloud/genesyscloud/validators"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	resourceExporter "terraform-provider-genesyscloud/genesyscloud/resource_exporter"
+	registrar "terraform-provider-genesyscloud/genesyscloud/resource_register"
+)
+
+const resourceName = `genesyscloud_externalcontacts_organization`
```

Comment:
We are changing our naming conventions. This should be `resourceType` now 

More details here - https://inindca.atlassian.net/browse/DEVTOOLING-924